init/systemd/cocos-agent.service

```diff
@@ -10,6 +10,7 @@ Environment=NETWORK_INTERFACE=enp0s3
 Environment=AGENT_GRPC_HOST=10.0.2.15
 Environment=AGENT_GRPC_PORT=7002
 Environment=AGENT_LOG_LEVEL=info
+Environment=COCOS_MESSAGE_BROKER_URL=192.168.100.4:4222
```

Comment:
What is this and why is this IP addr chosen?

Reply:
It is the host ip, this is specific in my case and could change for different users